### PR TITLE
chore: Update USWDS template metadata

### PR DESCRIPTION
**Pull request by apburnes**
## NOTICE:
- Relies on https://github.com/cloud-gov/cg-provision/pull/1134 to merge this PR

## Changes proposed in this pull request:
- chore: Update 11ty template metadata to current USWDS version
- chore: Update all template metadata  urls to custom domains `uswds-<template engine>.pages.cloud.gov`
- chore: Update 11ty thumbnail

## security considerations
none

### CODE REVIEW

File: config/templates.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ module.exports = {
     repo: 'pages-uswds-11ty',
     branch: 'main',
     engine: 'node.js',
-    example: 'https://federalist-32bdb3ca-a71a-46e5-ab20-a38ecfdbf143.app.cloud.gov/site/cloud-gov/pages-uswds-11ty/',
-    title: '11ty - U.S. Web Design System v2.0',
+    example: 'https://federalist-32bdb3ca-a71a-46e5-ab20-a38ecfdbf143.sites.pages.cloud.gov/site/cloud-gov/pages-uswds-11ty/',
+    title: '11ty - U.S. Web Design System v3.0',
     description: 'A flexible multi-page setup intended to introduce an agency or program. It offers an engaging Hero section for a current priority and a landing page layout designed to provide clarity and context for first time visitors. This template also offers navigation options for internal pages.',
     thumb: '/images/pages-uswds-v2-thumbnail.png',
```
Comment (drewbo) [inline config/templates.js:10]: Should we rename and/or recreate the thumbnail?

**Review by drewbo — CHANGES_REQUESTED** [review]

File: config/templates.js (hunk 1 above)
Comment (apburnes) [reply config/templates.js:10]: Probably. Also, should we create custom domains for these?

Comment (apburnes) [reply config/templates.js:10]: ie. `uswds-11ty.pages.cloud.gov`

Comment (apburnes) [reply config/templates.js]: Updated. Will wait on https://github.com/cloud-gov/cg-provision/pull/1134 before merging.

**Comment by drewbo** [comment]
@apburnes Looks like the custom URLs aren't resolving yet but we can merge once they do

**Review by drewbo — DISMISSED** [review]

File: config/templates.js [hunk 2]
```diff
@@ -4,17 +4,17 @@ module.exports = {
     repo: 'pages-uswds-11ty',
     branch: 'main',
     engine: 'node.js',
-    example: 'https://federalist-32bdb3ca-a71a-46e5-ab20-a38ecfdbf143.app.cloud.gov/site/cloud-gov/pages-uswds-11ty/',
-    title: '11ty - U.S. Web Design System v2.0',
+    example: 'https://uswds-11y.pages.cloud.gov',
```
Comment (drewbo) [inline config/templates.js:7]:
```suggestion
    example: 'https://uswds-11ty.pages.cloud.gov',
```

**Review by drewbo — APPROVED** [review]

**Pull request merged**